src/restore/cross_account_db_cluster_restore.py

```diff
@@ -0,0 +1,72 @@
+import os
+import boto3
+import constants
+import custom_exceptions
+import utility as util
+
+def lambda_restore_rds_cluster_target_account(event, context):
+    region = os.environ['Region']
+    ssm = boto3.client('ssm', region)
+    rds = boto3.client('rds', region)
+    result = {}
+    try:
+        describe_cluster_response = rds.describe_db_clusters(
+            DBClusterIdentifier=event['identifier']
+        )
+        engine = describe_cluster_response['DBClusters'][0]['Engine']
+        engine_version = describe_cluster_response['DBClusters'][0]['EngineVersion']
+        db_cluster_members = []
+        db_cluster_instance_class = []
+        for db_cluster_member in describe_cluster_response['DBClusters'][0]['DBClusterMembers']:
```

Comment:
Can we use list comprehensions to avoid a for loop . Something like 
```
db_cluster_members = [db_cluster_member['DBInstanceIdentifier'] for db_cluster_member in describe_cluster_response['DBClusters'][0]['DBClusterMembers']]
db_cluster_instance_class = [desc_db_response['DBInstances'][0]['DBInstanceClass'] for desc_db_response in [rds.describe_db_instances(DBInstanceIdentifier=db_cluster_member['DBInstanceIdentifier']) for db_cluster_member in describe_cluster_response['DBClusters'][0]['DBClusterMembers']]]
```